README.md

```diff
@@ -305,14 +308,17 @@ Override `globals` object in Jest config:
       "ts-jest": {
         "tsConfig": "<rootDir>/src/tsconfig.custom.json",
         "stringifyContentPathRegex": "\\.html$",
-        "astTransformers": ["jest-preset-angular/InlineHtmlStripStylesTransformer"],
+        "astTransformers": [
+          "jest-preset-angular/build/InlineFilesTransformer",
+          "jest-preset-angular/build/StripStylesTransformer"
+        ],
       }
     }
   }
 }
 ```
 
-If you choose to overide `globals` in order to point at a specific tsconfig, you will need to add `"astTransformers": ["jest-preset-angular/InlineHtmlStripStylesTransformer"]` to the `globals.ts-jest` section too, otherwise you will get parse errors on any html templates.
+If you choose to overide `globals` in order to point at a specific tsconfig, you will need to add the `astTransformers` to the `globals.ts-jest` section too, otherwise you will get parse errors on any html templates.
```

Comment:
How about linking to the specific lines in the source file?

Reply:
The *Custom tsconfig* with the full example part is just right above this line, I think that is easy enough to copy. If we link the full preset, they might copy wrong parts, like `"astTransformer": ["./build/Transformer"]`, so I actually prefer this style.

Plus the configuration is linked on the very top of the README.

Reply:
![image](https://user-images.githubusercontent.com/20015207/65030343-a8393800-d93f-11e9-9049-d9187d92c28f.png)